public/assets/css/article.css

```diff
@@ -1,38 +1,88 @@
 article {
     color: black;
+    margin: 20px;
+}
+
+h1 {
+    font-size: 1.49rem;
+    font-weight: 900;
 }
 
 h2 {
     color: royalblue;
 }
 
-body main>article header h2 {
-    color: gray;
+ul {
+    white-space: nowrap;
+    text-align: center;
+}
+
+li {
+    display: inline;
 }
 
 img {
-    background-image: url("/assets/images/image_clavier_anisse.png");
+    background-image: url("./assets/images/image_clavier_anisse.png");
 }
 
 pre {
     border: 3px solid dodgerblue;
 }
 
-main footer {
+blockquote {
+    border-left: 10px solid lightgrey;
+    border-right: 0.5px solid lightgrey;
+    width: 380px;
+    border-block-start: 0.5px solid lightgrey;
+    border-block-end: 0.5px solid lightgrey;
+    border-start-start-radius: 3px;
+    border-bottom-left-radius: 3px;
+}
+
+body>footer {
```

Comment:
no 
```css
main>footer
```

because otherwise you will target the website footer

and if you want to target the footer of website please would create a new file `footer.css`